consensus/src/experimental/tests/commit_phase_tests.rs

```diff
@@ -109,17 +121,26 @@ fn prepare_commit_phase() -> (
     let network = NetworkSender::new(author, network_sender, self_loop_tx, validators);
 
     let (commit_result_tx, commit_result_rx) =
-        channel::new_test::<(Vec<Block>, LedgerInfoWithSignatures)>(channel_size);
+        channel::new_test::<ExecutionChannelType>(channel_size);
     let state_computer = Arc::new(OrderingStateComputer::new(commit_result_tx));
 
+    let time_service = Arc::new(ClockTimeService::new(runtime.handle().clone()));
+
+    let block_store = Arc::new(BlockStore::new(
```

Comment:
don't need?

Reply:
we need this in the integration test. Maybe we can remove block_store insertions in commit_phase_test

Reply:
hmm, that should be a util function in integration test which wraps this one?

Reply:
resolved